build.sh

```diff
@@ -143,11 +149,12 @@ pixman_dependencies=""
 cairo_dependencies="zlib png pixman"
 xml_dependencies="zlib iconv"
 sqlite_dependencies=""
-openslide_dependencies="png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
+openslide_dependencies="libzip png jpeg tiff openjpeg glib gdkpixbuf cairo xml sqlite"
 openslidejava_dependencies="openslide"
 
 # Build artifacts
 zlib_artifacts="zlib1.dll"
+libzip_artifacts="libzip.dll"
```

Comment:
This sounds great. I thought I had pushed it. Now I have to wait till Wednesday when I'm back home and then I can reply to the other feedback then, as well.
But this answers the other problem I had in favour of zip_open() - which I actually prefer myself, too. Let the library choose what to do.